Shared/API/Onboarding+Cloud.swift

```diff
@@ -66,15 +68,38 @@ struct VerifyCode: CloudFunction {
 }
 
 struct ActivateUser: CloudFunction {
-    
+
     typealias ReturnType = Any
+    let fullName: String
 
     @discardableResult
     func makeRequest(andUpdate statusables: [Statusable], viewsToIgnore: [UIView]) async throws -> Any {
+
+        let fullName = self.formatName(from: fullName)
```

Comment:
We now pass the full name in here vs saving to the user before hand. Saves a call.